Artifacts/windows-azurepowershell/StartChocolatey.ps1

```diff
@@ -0,0 +1,56 @@
+Param(
+    #
+    [ValidateNotNullOrEmpty()]
+    $ProductId
+)
+
+Function Get-TempPassword() {
+    Param(
+        [int]$length=10,
+        [string[]]$sourcedata
+    )
+
+    For ($loop=1; $loop �le $length; $loop++) {

```

Comment:
There seems to be a difference between the versions of this file in each of the artifacts in this PR. For example, in this usage, the '''For''' loop refers to '''$length''' but in other versions, '''$length''' has been replace by a magic number '''126'''